cognitive_architecture/database/relationaldb/database.py

```diff
@@ -0,0 +1,92 @@
+"""Database configuration and connection."""
+from pathlib import Path
+from contextlib import asynccontextmanager
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+from sqlalchemy.orm import declarative_base, sessionmaker
+from dotenv import load_dotenv
+from cognitive_architecture.config import Config
+
+
+load_dotenv()
```

Comment:
Do we need to call this function everywhere? I think that calling it once in the app would be fine.

Reply:
In the config?

Reply:
Not sure. Maybe somewhere in the code before we start the server.
Some part that is executed first.